modules/client-configuration/pages/clients-slmicro.adoc

```diff
@@ -24,9 +24,29 @@ include::snippets/arch-other-note.adoc[]
 
 The products you need for this procedure are:
 
-[[slmicro-channels-wizard]]
+
+[[slmicro-channels-wizard-61]]
+[cols="1,1", options="header"]
+.{sl-micro} {microversion} Products - WebUI
+|===
+
+| OS Version
+| Product Name
+
+| {sl-micro} 6.1 {x86_64}
+| SUSE Linux Micro 6.1 x86_64
+
+| {sl-micro} 6.1 {arm64}
+| SUSE Linux Micro 6.1 arch64
+
+| {sl-micro} 6.1 {s390x}
+| SUSE Linux Micro 6.1 s390x
+
+|===
+
+[[slmicro-channels-wizard-60]]
 [cols="1,1", options="header"]
-.SL Micro Products - WebUI
+.{sl-micro} 6.0 Products - WebUI
```

Comment:
Maybe, better 6.1 or just 6 here?

Reply:
I have two small sections. One for the default 6.1 and the other for 6.0.